CONTRIBUTING.md

```diff
@@ -13,7 +13,11 @@ section.
 ## Modules
 
 * Prefer the use of the singular form in module name (or use "multi")
-* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module)
+* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module) but remove sections with no meaningful content.
```

Comment:
I stand in that text and believe it's good for it to be added.
Anyone disagreeing can discuss it here.